src/validation/errorMessage.js

```diff
@@ -0,0 +1,21 @@
+import { validationResult } from 'express-validator/check';
+
+const Errs = {
```

Comment:
This variable name isn't descriptive enough. Can we use `Errors` or something else that'll easily tell what the object is?

Reply:
ok thank, have implemented it